lib/cosmos/script/script.rb

```diff
@@ -77,6 +68,28 @@ def method_missing(method_name, *method_params)
       when :disconnect
         @cmd_tlm_server.disconnect
       else
+        if $disconnect_all_targets
+          return @disconnected.send(method_name, *method_params)
+        elsif $disconnected_targets
+          name_string = nil
+          if method_params[0].is_a?(String)
+            name_string = method_params[0]
+          elsif method_params[0].is_a?(Array)
```

Comment:
Is this to capture the alternative syntax like
```
cmd("INST", "COLLECT", "DURATION" => 10, "TYPE" => "NORMAL")
```

Reply:
No, it is to handle methods where the first parameter is an array like get_tlm_details